docs/development.md

```diff
@@ -34,62 +34,13 @@ Still in Terminal, run `yarn install`. This will download and install all the de
 
 ### Viewing the site locally
 Start server by running `npm run start`.
-  * If `npm run start` fails, run ```$ npm i```. Then do ```$ npm run start``` again.
+  * If `npm run start` fails, run `$ npm i`. Then do `$ npm run start` again.
   * If the `fractal watch` task fails, remove the `dist` directory and try running `npm run start` again.
 
 Stop server by pressing *CTRL + c* keys while you're in Terminal.
 
 View the site by navigating to `localhost:3002` in your favorite browser.
 
-
```

Comment:
I removed this stuff because it seemed irrelevant to this repo. I think it was copy-pasted from another project.

Reply:
It looks like this is from the design system (formerly named `jean-pants`), could we simply correct the name here or are the steps now different?

Reply:
I don't think these publication steps apply to this repo at all, since this one is just published via gh-pages. But @rtluu would have a better idea. Ryan, do you think it's safe to delete these instructions? It looks as though this repo started as a copy of the jean-pants/design system repo and this stuff doesn't apply to these platform docs.

Reply:
Thanks @erikphansen, should we replace these steps as well as remove them?

Reply:
I'm not sure there's anything to replace these instructions with. The instructions above are correct and valid, but all this "module publishing" stuff is not needed, as far as I can tell.